File: Python/Practices/financial_calculator.py
# IB Finacial Calculator - Python

income = round(float(input("What is your monthly income: $ ")), 2)

rent = round(float(input("What is your monthly rent/mortgage: $ ")), 2)

utilities = round(float(input("What are your monthly utilities: $ ")), 2)

groceries = round(float(input("What is your monthly groceries: $ ")), 2)

transportation = round(float(input("What is your monthly transportation: $")), 2)

percent = round(float(input("What percentage of your income would you like to save? (Exclude the percentage symbol) It is recomended 10% and higher: ")), 2)

percent_of_savings = percent/100

savings = round(float(income*percent_of_savings), 2)

spending = round(float(income-rent-utilities-groceries-transportation-savings), 2)



print("Your rent is $", rent, "and that is", round((rent/income)*100), "%","of your income")

print("Your utilities are $", utilities,"and that is", round((utilities/income)*100), "%", "of your income")

print("Your Groceries are $", groceries, "and that is", round((groceries/income)*100), "%", "of your income")

print("Your Transportation is $", transportation, "and that is", round((transportation/income)*100), "% of your income")

print("You should save $", savings,"a month, and that is", int(percent), "% of your income" )

print("You have $", spending, "of sepending money each month!")

File: C/Practices/financial_calculator.c
// IB 7th Financial Calculator
#include <stdio.h>

int main(){

    float income; 
    printf("What is your monthly income: $");
    scanf("%f", &income);

    float rent;
    printf("What is your monthly rent/mortgage: $");
    scanf("%f", &rent);

    float utilities;
    printf("What is your monthly utilities: $");
    scanf("%f", &utilities);

    float transportation;
    printf("What is your monthly transportation: $");
    scanf("%f", &transportation);

    float groceries;
    printf("What is your monthly groceries: $");
    scanf("%f", &groceries);

    float savings = income * 0.10;
    float totalExpenses = rent + utilities + groceries + transportation;
    float spendingMoney = income - totalExpenses - savings;


    printf("\nYour rent is $%.2f and that is %.0f%% of your income.\n", rent, (rent/income)*100);

    printf("Your utilities are $%.2f and that is %.0f%% of your income.\n", utilities, (utilities/income)*100);

    printf("Your groceries are $%.2f and that is %.0f%% of your income.\n", groceries, (groceries/income)*100);

    printf("Your transportation is $%.2f and that is %.0f%% of your income.\n", transportation, (transportation/income)*100);

    printf("You should save $%.2f a month, that is 10%% of your income.\n", savings);

    printf("You have $%.2f of spending money each month!\n", spendingMoney);
    

    return 0;
}